server/neptune/workflows/internal/activities/store.go

```diff
@@ -6,20 +6,21 @@ import (
 	"github.com/runatlantis/atlantis/server/neptune/logger"
 )
 
-type dbActivities struct {
+type storeActivities struct {
 }
 
 type FetchLatestDeploymentRequest struct {
-	RepositoryURL string
-	RootName      string
+	RepositoryName string
+	RootName       string
 }
 
 type FetchLatestDeploymentResponse struct {
 	ID       string
 	Revision string
 }
 
```

Comment:
Request/Response is most likely going to change when this is implemented, this is just added to build the skeleton for now.